R/utils.R

```diff
@@ -0,0 +1,32 @@
+fromRJSON <- function(x, ...) {
```

Comment:
lintR keeps yelling at me for not using lower_snake_case so that's what I've been using in this project & the addin so far. Maybe we should do some sort of consistent styling checks using CI and whatnot, but I think for now, trying to keep it consistent manually makes sense

Reply:
Sure, I'll rename it. The reason I used camelCase here is that jsonlite also uses it and since `fromRJSON` is just a small wrapper around `fromJSON` it seemed natural to also mimic the wrapped functions naming style.

Reply:
Free for grabs :D (https://github.com/flowr-analysis/flowr-r-adapter/issues/3).